### ISSUE TITLE
Cannot Save Field In Template (PW Dev 2.5.22)

### ISSUE THREAD

**Issue by cstevensjr** [ryancramerdesign/ProcessWire#999]
When trying to add a field to an existing template, when you hit Save the field is not saved.

Figure 1 - Template Before Trying to add a new field

![pw-template-field-add1](https://cloud.githubusercontent.com/assets/7488817/6540000/a9781ab4-c43c-11e4-9d9d-10ca9b28af71.png)

Figure 2 - Template Showing new field to be added

![pw-template-field-add2](https://cloud.githubusercontent.com/assets/7488817/6540006/ca84c6ee-c43c-11e4-98ce-11fdfcb5ff8c.png)

**Comment by adrianbj** [comment]
Hey Charles - I can't seem to duplicate this - was it working before 2.5.22?

What browser and version?

Does it consistently fail, or only occasionally ?

**Comment by teppokoivula** [comment]
Additionally: do you have any 3rd party modules installed, and do you see anything strange in your Apache or PW (error) log files? Any chance you could try this with a fresh installation on the same environment and tell us if that works? :)

**Comment by cstevensjr** [comment]
Now it's working.  I can't explain what happened before as I tested the problem at least 3 times before opening this issue.    I'm using Google Chrome on an iMac.

Sorry for bringing this up, I will test on my many other additional computers in the future before opening an issue.

**Comment by adrianbj** [comment]
No problem at all - it sounds like there might be an issue, even if sporadic. Just to clarify, is this just adding an existing field to the template, or is it the new "create a new field functionality"? If the latter, does the field actually get created, but is just not added to the template?

**Comment by cstevensjr** [comment]
Just adding an existing field.

Normally, I would have checked this out in another browser and on another computer.  This incident just reinforces what I already knew -- to be very thorough in my analysis and problem resolution.

I should have been extra careful anyway, as I had recently installed new software on this iMac.  I also had for the first time connected my iPad to the iMac to record some videos.  I had noticed some connectivity issues with my bluetooth mouse after I made the connection to the iPad.  This was around the same time I started working with PW and experienced the problem.

Like I said before, I know I experienced it because it surprised me when it happened.  I then went to work on other tasks I had. When I saw the responses of you and Teppo, I brought up the website in Firefox and was able to add the field to the template.  I should have tried that process before opening the issue.  Then I went back to Chrome and it was working again.

I really don't have an answer, however I'm just glad it's working and not a PW problem.
